libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFSDKAuthPreferences.m

```diff
@@ -217,7 +217,18 @@ - (void)setIsIdentityProvider:(BOOL)isIdentityProvider
 - (NSString *)appDisplayName
 {
     NSUserDefaults *defs = [NSUserDefaults msdkUserDefaults];
-    return [defs stringForKey:kOAuthAppName];
+    NSString *appName = [defs stringForKey:kOAuthAppName];
+    
+    if (appName) {
+        return appName;
+    } else {
+        NSString *bundleDispalyName = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleDisplayName"];
```

Comment:
@brandonpage Typo on the variable - should be `bundleDisplayName` (it is `bundleDispalyName`).